eng/testing/tests.props

```diff
@@ -8,6 +8,7 @@
     <VSTestNoLogo>true</VSTestNoLogo>
     <ILLinkDescriptorsPath>$(MSBuildThisFileDirectory)ILLinkDescriptors\</ILLinkDescriptorsPath>
     <TestSingleFile Condition="'$(TestNativeAot)' == 'true'">true</TestSingleFile>
+    <TestSingleFile Condition="'$(TestCrossgen2)' == 'true'">true</TestSingleFile>
```

Comment:
Would it make sense to just name this TestReadyToRun?

Reply:
Yes. That's a better name